02-peer-protocol.md

```diff
@@ -559,6 +560,8 @@ A sending node:
         (witness program versions 1 through 16)
 
 A receiving node:
+  - if it receives multiple `shutdown` messages:
+    - SHOULD ignore all except the first one.
```

Comment:
Comment from spec meeting: should change to say you can only send one `shutdown` per reconnection. 

The receiver can do w/e they want if this is violated technically.

Reply:
@vincenzopalazzo if you remove these receiver requirements, I believe we can merge this as discussed during the last spec meeting?

Reply:
Yes! I will refine this PR by the end of the week